cds_ils/importer/providers/cds/helpers/parsers.py

```diff
@@ -208,13 +208,19 @@ def _clean(value_to_clean):
             return default
         raise MissingRequiredField(subfield=subfield)
 
-    if multiple_values and type(to_clean) is tuple:
-        cleaned_values = []
-        for v in to_clean:
-            cleaned_values.append(_clean(v))
-        return cleaned_values
-    elif not multiple_values and type(to_clean) is tuple:
+    is_tuple = type(to_clean) is tuple
+    if is_tuple and not multiple_values:
         raise UnexpectedValue(subfield=subfield)
+
+    if multiple_values:
```

Comment:
changed the return value to be always a `list`, and changed all usage of this func.